src/components/network-chart/NetworkChartRow.js

```diff
@@ -34,7 +34,12 @@ export type NetworkChartRowBarProps = {
   +marker: TracingMarker,
   // Pass the payload in as well, since our types can't express a TracingMarker with
   // a specific payload.
-  +networkPayload: NetworkPayload,
+  markerMeta: {
```

Comment:
I think this `markerMeta` name is a bit ambiguous. Could it be renamed to something like `networkRequestInfo`?